Contest_API.md

```diff
@@ -164,36 +164,31 @@ fail or implement a cascading delete (to maintain [referential integrity](#refer
 When there is a failure using any method the response message body
 must include a JSON object that contains the properties 'code' (a number,
 identical to the HTTP status code returned) and 'message' (a string) with
-further information suitable for the user making the request, as per the
+further information suitable for the client making the request, as per the
 following example:
 
 ```json
 {"code":403,
  "message":"Teams cannot send clarifications to another team"}
  ```
 
-### Roles
+### Authentication
 
-Access to this API is controlled via user roles. The API provider must
-require authentication to access each role except for optionally the
-public role. The API provider must support [HTTP basic
+The API provider may allow unauthenticated access for information
+that is fully public - i.e. may be visible to everyone including
+spectators and teams. If provided this must be read-only access
+(no `POST`, `PUT`, `PATCH` or `DELETE` methods allowed).
+
+All other access to the API must be controlled via
+authenticated accounts. The API provider must support [HTTP basic
 authentication](https://en.wikipedia.org/wiki/Basic_access_authentication)
 ([RFC](https://tools.ietf.org/html/rfc7617)). This provides a standard
 and flexible method; besides HTTP basic auth, other forms of
 authentication can be offered as well.
 
-Each provider must support at least the following roles, although
-additional roles may be supported for specific uses:
-
-  - public (default role: contest data that's available to everyone)
-  - admin (data or capability only available to contest administrators)
-
-Role-based access may completely hide some objects from the user, may
-omit certain properties, or may embargo or omit objects based on the
-current contest time. By default, the public user has read-only access
-(no `POST`, `PUT`, `PATCH` or `DELETE` methods allowed) and does
-not have access to judgements and runs from submissions made after the
-contest freeze time.
+Depending on the client account's access, some objects may be completely
+hidden from the client, may omit certain properties, or may embargo or
+omit objects based on the current state of the contest.
```

Comment:
Small rewrite to active voice and with consistent subject/object:
```suggestion
Depending on the client account's access, the API provider may completely
hide some objects from the client, may omit certain properties, or may embargo or
omit objects based on the current state of the contest.
```

Reply:
I actually don't think the active voice is better here. It's not that the server after careful consideration decides to hide some objects, but rather that given the capabilities (I almost said role) of the client and the state of the contest, some objects are (and should be) hidden.

Reply:
FYI, my reason for using the active voice was mostly to make the sentence shorter, and I think, slightly easier to read. It's not meant to change any intented meaning/intonation. Also the original sentence was still written half/half in active/passive voice. I'm also fine with completely writing it in passive voice.

Reply:
I'm going to commit just b/c it makes the wording consistent. If anyone disagrees we can change the whole thing to passive voice.